### PR TITLE
Extend range of Maestro cards IIN

### PR DESCRIPTION
**Pull request by antondozortsev-zz**

### CODE REVIEW

File: lib/angular-payments.js [hunk 1]
```diff
@@ -135,7 +126,16 @@ angular.module('angularPayments', []);angular.module('angularPayments')
       length: [16, 17, 18, 19],
       cvcLength: [0, 3],
       luhn: true
-    }
+    }, {
+      type: 'maestro',
+      brand: 'maestro',
+      pattern: /^(5018|5020|5038|6)/,
```
Comment (antondozortsev-zz) [inline lib/angular-payments.js:132]: Pattern updated according wiki: https://en.wikipedia.org/wiki/Payment_card_number#Issuer_identification_number_.28IIN.29

Comment (FatimehinDare) [reply lib/angular-payments.js:132]: You tested it in staging?

Comment (antondozortsev-zz) [reply lib/angular-payments.js:132]: yes, https://delphinus.transferwise.com/

Comment (FatimehinDare) [reply lib/angular-payments.js:132]: 👌

Comment (antondozortsev-zz) [reply lib/angular-payments.js:132]: Tested using those Maestro cards: https://support.worldpay.com/support/kb/bg/testandgolive/tgl5103.html

and: https://creditcall.zendesk.com/hc/en-gb/articles/115002375583-Test-Cards

**Pull request merged**